runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -3030,6 +3043,7 @@ class INTERPRETER_CLASS
 							buildInternalNativeStackFrame(REGISTER_ARGS);
 							rc = THROW_ARRAY_STORE;
 						} else {
+
```

Comment:
Please remove the empty line change.

Reply:
fixed in [92cb616](https://github.com/eclipse-openj9/openj9/commit/92cb6169f6ef83030ce3d192917cf7b963164ba2)